notebooks/official/tensorboard/tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,968 @@
+{
```

Comment:
<p>The steps performed include the following: </p><p> </p><ul><li>Set up a service account and a Google Cloud Storage bucket.</li><li>Write your customized training code.</li><li>Package and upload your training code to the Cloud Storage.</li><li>Create and launch your custom training job with Tensorboard enabled for near real time monitoring.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>